ml4gw/spectral.py

```diff
@@ -388,12 +393,16 @@ def truncate_inverse_power_spectrum(
     # impulse response function
     inv_asd = 1 / psd**0.5
 
-    # zero our leading frequencies if we want the
-    # filter to perform highpass filtering
+    # zero out frequencies if we want the filter
+    # filter to perform highpass/lowpass filtering
     if highpass is not None:
         df = sample_rate / N
```

Comment:
maybe define this `df` outside of the if statements to be used for both. Also, minor typo in the comment above, looks like theres an extra "filter"